projects/js-packages/jetpack-cli/bin/jp.js

```diff
@@ -170,7 +170,7 @@ const main = async () => {
 					fs.mkdirSync( resolve( monorepoRoot, 'tools/docker/wordpress' ), { recursive: true } );
 
 					// Create empty .env file
-					fs.writeFileSync( resolve( monorepoRoot, 'tools/docker/.env' ), '' );
+					fs.writeFileSync( resolve( monorepoRoot, 'tools/docker/.env' ), '', { flag: 'a' } );
```

Comment:
I note `tools/cli/commands/docker.js` does like this instead
```suggestion
					fs.closeSync( fs.openSync( resolve( monorepoRoot, 'tools/docker/.env' ), 'a' ) );
```
i.e. it opens and closes the file with no write. Probably that doesn't really make a difference though. 🤷